packages/client/src/components/CommentCard.vue

```diff
@@ -3,7 +3,7 @@
     <div class="wl-user" aria-hidden="true">
       <img v-if="comment.avatar" :src="comment.avatar" />
 
-      <VerifiedIcon v-if="comment.type" />
+      <VerifiedIcon v-if="comment.type === 'administrator'" />
```

Comment:
What’s this?

Reply:
I am so sorry about this, this was a push error.